tests/gis_tests/layermap/tests.py

```diff
@@ -324,6 +325,20 @@ def allow_relation(self, obj1, obj2, **hints):
     def allow_migrate(self, db, app_label, **hints):
         return True
 
+    def test_null_geom_with_unique(self):
+        """ Test a null geom field, with unique keyword argument """
+        provinces_shp = os.path.join(
```

Comment:
Use a single line. Our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style) allows lines up to 119 characters if it helps readability.